### ISSUE TITLE
Unpublish request `ms-pyright.vscode-pyright`@`1.1.287`

### ISSUE THREAD

**Issue by filiptronicek** [EclipseFdn/open-vsx.org#1532]
Because there are some troubles with VS Code's Pyright extension, we need to re-publish the latest release of the extension, which should be [`1.1.287`](https://open-vsx.org/extension/ms-pyright/pyright/1.1.287). 

There is a [bug](https://github.com/microsoft/pyright/issues/4416) present because of our resolution logic, which should be now fixed. 

@kineticsquid would you be able to help here? 🙏

**Comment by kineticsquid** [comment]
@amvanbaren Could you help here? I'm attempting to pull up this extension on the admin panel. I can get to it from publisher or namespace, but when I attempt to get it from the extension link, the page keeps spinning and never returns with results.

**Comment by amvanbaren** [comment]
Extension not found: https://open-vsx.org/api/ms-pyright/vscode-pyright
@filiptronicek Do you mean `ms-pyright.pyright`? https://open-vsx.org/api/ms-pyright/pyright/1.1.287

**Comment by kineticsquid** [comment]
@amvanbaren Yes, I assumed that was the extension @filiptronicek was referring to. I can see the extension in the admin panel by searching the namespace `ms-pyright`. I see one extension, `ms-pyright.pyright`. But when I use the extension link to search for that specific extension, the page never returns with the result.

**Comment by filiptronicek** [comment]
Sorry @kineticsquid, yes, as @amvanbaren pointed out, it's `ms-pyright.pyright`. One of the issues with the extension is that Microsoft changes the ID for the extension when publishing so there are some issues in there.

**Comment by kineticsquid** [comment]
I was able to reproduce this in a deployment I have in a gitpod workspace. I successfully published the extension, I can see it from the admin panel when searching by publisher (me) or by namespace. But when I search for it from the `Extension` link on the admin panel, same behavior, the search never returns. 

Also, it's not displayed on the main page with all the other plugins. Though I see some weird behavior from time to time where this page shows an incomplete list. And sometimes when I publish an extension, it doesn't show up right away. Is there some kind of caching going on?

@filiptronicek are you saying the extension file is corrupt in some way?
